src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -135,12 +136,18 @@ abstract class IntegrationTestBase {
   }
 
   fun getTimeline(prisonNumber: String): TimelineResponse =
-    webTestClient.get()
-      .uri(GET_TIMELINE_URI_TEMPLATE, prisonNumber)
-      .bearerToken(aValidTokenWithViewAuthority(privateKey = keyPair.private))
-      .exchange()
-      .returnResult(TimelineResponse::class.java)
-      .responseBody.blockFirst()!!
+    run {
+      wiremockService.stubGetPrisonTimelineFromPrisonApi(
```

Comment:
Minor fix here to stub the call to prison API to get an empty set of prison movements for the prisoner
This is because the call to our `GET /timelines/{prisonNumber}` (ie. what the web test client is doing) calls prison api to get prisoner movements, and also queries our timeline database.
Without the stub to prison api, whilst it didnt break any tests because we have suitable error handling in the timeline service class, it did result in quite a lot of log noise when the tests run